pkg/scheduler/framework/v1alpha1/interface.go

```diff
@@ -155,6 +155,39 @@ func NewStatus(code Code, reasons ...string) *Status {
 	}
 }
 
+type PluginToStatus map[string]*Status
+
+func (p PluginToStatus) Merge() *Status {
```

Comment:
comments missing.

We should clarify the intent. We only seemed concerned about Filter status for now. Is that enough?

Reply:
added comments.